src/store/actions/description.ts

```diff
@@ -0,0 +1,18 @@
+import { DescriptionAction, DescriptionActionTypes } from '../redusers/types'
+import { Dispatch } from 'redux'
+
+export const fetchDescription = (param: any) => {
```

Comment:
please read this on how to create actions https://react-redux.js.org/tutorials/typescript-quick-start#define-slice-state-and-action-types